### PR TITLE
DE-692 Remove reference to insecure identify response fields

### PR DESCRIPTION
**Pull request by mnadel**
https://dripcom.atlassian.net/browse/DE-692

### CODE REVIEW

File: source/includes/js/_identify.md [hunk 1]
```diff
@@ -82,9 +82,6 @@ The following are treated as special properties. All other data passed in will b
   success: true,
   visitor_uuid: "f627ee608adb01315d1022000ab2058a",
   anonymous: false,
-  email: "john@acme.com",
-  custom_fields: { "name": "John" },
-  tags: ["Customer"],
   lead_score: 35
```
Comment (andrewstucki) [inline source/includes/js/_identify.md]: this is gone too:

```
    @payload = {
      success: true,
      visitor_uuid: @visitor_uuid,
      anonymous: @identity.anonymous?
    }
```

only those remain.

Comment (mnadel) [reply source/includes/js/_identify.md]: Sorry it's not clear - what's else did we remove, @andrewstucki?

Comment (andrewstucki) [reply source/includes/js/_identify.md]: sorry, `lead_score` shouldn't be on there either

Comment (mnadel) [reply source/includes/js/_identify.md]: Ah! Thanks.

**Pull request merged**